components/ellipsis/ellipsis.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <div
     ref="wrapper"
-    :class="nh.b()"
+    :class="props.maxLines > 0 ? nh.bm('multiple') : nh.b()"
```

Comment:
`nh.b()` is necessary, it ensure components have a basis class name all the time.

It can be changed to `:class="[nh.b(), props.maxLines > 0 && nh.bm('multiple')]"`